scripts/allocate-compute.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+
+import argparse, sys, time
+
+import common, connection
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description='Allocate storage and assign it to a coumpte node',
+        epilog = 'Must executed on Rabbit with nnf-ec running')
+    
+    parser.add_argument('--node', nargs='+', choices=range(0,16), default=None, help='the node index to attach to')
```

Comment:
How would a user know that 'None' means 'All'? Wouldn't the default be 'all'?

Reply:
I updated the usage